python_scripts/functions.py

```diff
@@ -269,11 +263,48 @@ def local_file_to_s3(local_path, s3_path):
     with open(local_path, "rb") as f:
         s3_client.upload_fileobj(f, b, o)
 
+
+
 def generate_iam_config(config, outpath="iam_config.yaml"):
     """
     Should take the necessary paths in the config and write out an
     iam that has access to those s3 paths e.g.
     write-only to log-base-path
     read/write to land-base-path
     """
-    pass
\ No newline at end of file
+
+    expected_keys = [
```

Comment:
Think the config schema should check this?

Reply:
very good point - the input to this function is an already-validated config file. I'll remove the redundant check